--- flyer-0.1/QPropertyEditor/Property.cpp~	2008-10-01 09:25:24.000000000 +0200
+++ flyer-0.1/QPropertyEditor/Property.cpp	2008-10-01 20:56:51.000000000 +0200
@@ -25,6 +25,7 @@
 
 #include <QtCore/QMetaProperty>
 #include <QtGui/QSpinBox>
+#include <limits.h>
 
 Property::Property(const QString& name /*= QString()*/, QObject* propertyObject /*= 0*/, QObject* parent /*= 0*/) : QObject(parent), 
 m_propertyObject(propertyObject)
